tests/devices/test_default_qubit_jax.py

```diff
@@ -24,6 +25,48 @@
 from pennylane.pulse import ParametrizedHamiltonian
 
 
+@pytest.mark.parametrize(
+    "version, package, should_raise",
+    [
+        ("0.4.4", jax, True),
+        ("0.4.3", jax, False),
+        ("0.4.4", jaxlib, True),
+        ("0.4.3", jaxlib, False),
+    ],
+)
+def test_jax_version(version, package, should_raise, monkeypatch):
```

Comment:
To be marked with Jax, otherwise test is skipped.

Reply:
🙈